packages/devextreme/js/ui/autocomplete.d.ts

```diff
@@ -162,6 +163,20 @@ export type OptionChangedEvent = EventInfo<dxAutocomplete> & ChangedOptionInfo;
  */
 export type PasteEvent = NativeEventInfo<dxAutocomplete, ClipboardEvent>;
 
+/**
+ * @docid _ui_autocomplete_SelectionChangingEvent
+ * @public
+ * @type object
+ * @inherits EventInfo,_ui_drop_down_editor_ui_drop_down_list_SelectionChangingInfo
+ */
+export type SelectionChangingEvent = EventInfo<dxAutocomplete> & SelectionChangingInfo & {
+    /**
+     * @docid _ui_autocomplete_SelectionChangingEvent.cancel
+     * @type Boolean|Promise<Boolean>
+     */
+     cancel: boolean | PromiseLike<boolean>;
```

Comment:
This argument is not optional because it's presented every time in the onSelectionChanging argument:
https://github.com/EugeniyKiyashko/DevExtreme/blob/24_2_onSelectionChanging_API/packages/devextreme/js/__internal/ui/collection/m_collection_widget.edit.ts#L464-L465